Dockerfile

```diff
@@ -37,7 +37,10 @@ RUN ln -sf /usr/local/julia/bin/julia /usr/local/bin/julia
 RUN useradd -m jupyter
 USER jupyter
 
-# Install Mira from github
+
+# Install Mira from github, with cache-bust
+ADD https://api.github.com/repos/DARPA-ASKEM/beaker-kernel/git/refs/heads/main bust-cache.json
```

Comment:
this `bust-cache.json` i think needs to exist and will need to change as well for the invalidation to work (ie version, date etc...)

Reply:
The file doesn't need to exist. The content of the file will change with each commit to beaker.

```
{"ref":"refs/heads/main","node_id":"REF_kwDOJgV4a69yZWZzL2hlYWRzL21haW4","url":"https://api.github.com/repos/DARPA-ASKEM/beaker-kernel/git/refs/heads/main","object":{"sha":"9be1fb2f9d1242321b140d501e6327d384195875","type":"commit","url":"https://api.github.com/repos/DARPA-ASKEM/beaker-kernel/git/commits/9be1fb2f9d1242321b140d501e6327d384195875"}}
```